server_addon/maya/server/settings/publishers.py

```diff
@@ -1171,6 +1174,8 @@ class PublishersModel(BaseSettingsModel):
         ]
     },
     "ValidateRenderSettings": {
+        "enabled": True,
+        "optional": False,
```

Comment:
This is missing the default value for the `active` setting @antirotor - you may have merged it too quickly?

Reply:
damn, the default value

Reply:
I've added it in #481